support-frontend/assets/pages/contributions-landing/components/PaymentMethodSelector.jsx

```diff
@@ -107,10 +111,19 @@ function withProps(props: PropTypes) {
   const fullExistingPaymentMethods: RecentlySignedInExistingPaymentMethod[] =
     ((props.existingPaymentMethods || []).filter(isUsableExistingPaymentMethod): any);
 
+  const legend = (
+    <legend className="form__legend">Payment method</legend>
+  );
+
+  const legendWithSecureTransaction = (
+    <div className="secure-transaction">
+      {legend} <SecureTransactionIndicator modifierClasses={['middle']} />
+    </div>
+  );
+
```

Comment:
To keep the logic consistent with how you did the logic elsewhere (outside of the return statement) you could consider an approach that is more like:

```
const legend = props.paymentSecurityDesignTestVariant === 'V2_securemiddle' ? 
(
    <div className="secure-transaction">
      {legend} <SecureTransactionIndicator modifierClasses={['middle']} />
    </div>
  ) :
(
    <legend className="form__legend">Payment method</legend>
  );
```

and then only render `legend` when the template is being returned

Reply:
Good shout re logic outside of template - I am using the original `legend` in both options though so will modify this slightly to make it work.

Reply:
yeah good shout!